### ISSUE TITLE
[0.9.2.2] Ignore Version Mismatch server option not working

### ISSUE THREAD

**Issue by MonzUn** [StrangeLoopGames/EcoIssues#20264]
When attempting to join a server that allows mismatched versions you are still blocked because the version is mismatching.
See screenshot below where the server is 0.9.2.1 and the client is 0.9.2.2.

![image](https://user-images.githubusercontent.com/11805935/106356498-43963780-6300-11eb-8a13-2070574823a4.png)

**Comment by mirasrael** [comment]
This option not intended for production usage. It is for development only, because in production it may lead to unpredictable behavior. So no client support, but if you really need then you can join server with Continue button or by ID/IP.

**Comment by MonzUn** [comment]
I am using it for mod development. DiscordLink doesn't care if it's a minor version off somewhere and I use my client for both my live server and the test server, so rollbacking or using the continue option is a hassle.

**Comment by mirasrael** [comment]
If you need to connect to development server then you can use hack I'm using :)
Set `version` in this file to current released server version:
Tools\Eco.VersionStampGit\Program.cs
like this
```csharp
var version       = isDebug ? $"{versionNumber} beta <debug>" : $"{versionNumber} beta {branch}-{suffix}";
            version = "0.9.2.2 beta release-149";
```

**Comment by IcedForge** [comment]
I would agree with Monzun on this it is not critical but it is A problem though and would like to see this fixed as i have the same issue when im fetching a github build, run the server and i cannot connect via join button and forced to use direct connect instead which when opening/closing repeatedly gets annoying after a while.

![Screenshot_133.jpg](https://images.zenhubusercontent.com/5f5a7fa59878ed957a861886/cac9849b-2f6b-4340-aed1-0bd9253849ae)
